commit 7527b67979d61609bff5bc8c54d2b11874a7bbfa
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:48 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index 197b34b..0ccad46 100644
--- a/configure.ac
+++ b/configure.ac
@@ -46,7 +46,7 @@ AC_SUBST(ATOBM_CFLAGS)
 AC_SUBST(ATOBM_LIBS)
 
 # bitmap checks for packages
-XAW_CHECK_XPRINT_SUPPORT(BITMAP)
+PKG_CHECK_MODULES(BITMAP, xaw7)
 PKG_CHECK_MODULES(XBITMAPS, xbitmaps)
 
 BITMAP_CFLAGS="$BITMAP_CFLAGS $XBITMAPS_CFLAGS"
